modules/skybrowser/src/wwtcommunicator.cpp

```diff
@@ -290,16 +291,34 @@ glm::dvec2 WwtCommunicator::equatorialAim() const {
 }
 
 void WwtCommunicator::setImageOrder(int i, int order) {
+    using it = std::deque<std::pair<int, double>>::iterator;
     // Find in selected images list
-    auto current = findSelectedImage(i);
-    auto target = _selectedImages.begin() + order;
+    it current = findSelectedImage(i);
+    int currentIndex = std::distance(_selectedImages.begin(), current);
+    int difference = order - currentIndex;
 
-    // Make sure the image was found in the list
-    if (current != _selectedImages.end() && target != _selectedImages.end()) {
-        // Swap the two images
-        std::iter_swap(current, target);
+    std::deque<std::pair<int, double>> newDeque;
+
+    for (int i = 0; i < _selectedImages.size(); i++) {
```

Comment:
This should yield a warning for comparing an int (`i`) with a `size_t` (`_selectedImages.size()`), that should be fixed before merging